ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -217,6 +209,14 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
+    private func setupCommandingLayout() {
+        if traitCollection.horizontalSizeClass == .regular && traitCollection.userInterfaceIdiom == .pad {
```

Comment:
I was about to ask how you'd handle being in compact mode on an iPad, but this looks like it should do the trick!